packages/react-devtools-core/src/standalone.js

```diff
@@ -208,7 +208,10 @@ function initialize(socket: WebSocket) {
     socket.close();
   });
 
-  store = new Store(bridge, {supportsNativeInspection: false});
+  store = new Store(bridge, {
+    supportsNativeInspection: false,
+    supportsReact: true,
```

Comment:
I think this param name is a bit confusing. React DevTools always "support React" :smile: We just want to detect an _unsupported version_.

I think this probably doesn't belong as a store capability.